.github/workflows/test.yml

```diff
@@ -17,14 +17,20 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      WG_NEO4J_HOST: 127.0.0.1
-      WG_NEO4J_PORT: 7687
-      WG_NEO4J_USER: neo4j
-      WG_NEO4J_PASS: testpass
       WG_COSMOS_HOST: ${{ secrets.WG_COSMOS_HOST }}
```

Comment:
Now that we have a local gremlin-based database for testing, can we remove the cosmos test db?

Reply:
I would suggest we keep Cosmos, as it uses some small variations in code paths within Warpgrapher, and very wildly different code paths within the gremlin-client library, because Microsoft is using two different outdated versions of the GraphSON wire format.  I'd suggest removing it if/when Microsoft ups their supported version of GraphSON to v3, which the rest of the world is using now.